config/configauth/go.mod

```diff
@@ -0,0 +1,54 @@
+module go.opentelemetry.io/collector/config/configauth
+
+go 1.19
+
+
+
```

Comment:
This go.mod looks strange now: perhaps add the direct dependencies first, the transitive after that, and the overrides at the bottom?

Reply:
I moved the replace, ptal